jsk_mbzirc_common/src/mbzirc_gazebo_truck_plugin.cpp

```diff
@@ -103,6 +107,25 @@ void GazeboTruck::Update()
   if ( terminated_ ) {
     return;
   }
+
+  //drone hit the ground check(200ms delay)
+  physics::ModelPtr drone_model = world_->GetModel("quadrotor");
+  if(!drone_model) return;
+  if(!takeoff_flag_)
+    {//TODO: check contact with model "arena" is better
+      if(landing_height_ < 0) landing_height_  = drone_model->GetWorldPose().pos.z;
+      if(drone_model->GetWorldPose().pos.z > landing_height_ + 0.05) //0.05m is a margin
+        takeoff_flag_ = true;
+    }
+  else
+    {//TODO: check contact with model "arena" is better
+      if(drone_model->GetWorldPose().pos.z < landing_height_ + 0.05) 
```

Comment:
@k-okada  A simple check of hitting ground by checking the relative height towards the ground.